deblur/test/test_parallel_deblur.py

```diff
@@ -0,0 +1,107 @@
+# -----------------------------------------------------------------------------
+# Copyright (c) 2015, The Deblur Development Team.
+#
+# Distributed under the terms of the BSD 3-clause License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# -----------------------------------------------------------------------------
+
+from unittest import TestCase, main
+from shutil import rmtree
+from tempfile import mkdtemp
+from os.path import join
+from glob import glob
+
+from bfillings.sortmerna_v2 import build_database_sortmerna
+
+from deblur.parallel_deblur import parallel_deblur
+
+
+class parallelDeblurTests(TestCase):
+    """Test parallel deblur methods functionality.
+    """
+    def test_parallel_deblur(self):
+        """Test parallel_deblur() functionality.
+        """
+        working_dir = mkdtemp()
```

Comment:
might be good to do this in `setUp` and have a corresponding remove in `tearDown`?